packages/admin/dashboard/src/components/data-grid/helpers/create-data-grid-price-columns.tsx

```diff
@@ -62,9 +66,9 @@ export const createDataGridPriceColumns = <
         type: "number",
         header: () => (
           <div className="flex w-full items-center justify-between gap-3">
```

Comment:
The icon is still cut off, so we should either change the wrapping div to a grid with `grid-cols-[1fr_15px] gap-3`, or add a `shrink-0` to the icon.